<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSearchLineWidget</class>
 <widget class="QWidget" name="CSearchLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLineEdit" name="find_LE"/>
     </item>
     <item>
      <widget class="QPushButton" name="find_next_PB">
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>29</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Find next object</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/liboaf_gui.qrc">
         <normaloff>:/icons/arrow_down.png</normaloff>:/icons/arrow_down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="find_prev_PB">
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>29</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Find previous object</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/liboaf_gui.qrc">
         <normaloff>:/icons/arrow_up.png</normaloff>:/icons/arrow_up.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>find_LE</tabstop>
  <tabstop>find_next_PB</tabstop>
 </tabstops>
 <resources>
  <include location="resources/liboaf_gui.qrc"/>
 </resources>
 <connections/>
</ui>
